### PR TITLE
2.x bind model

### PR DESCRIPTION
**Pull request by bancer**

### CODE REVIEW

File: Lib/Shim.php [hunk 1]
```diff
@@ -22,6 +22,13 @@ class Shim {
 	 */
 	const VALIDATE_PROPERTY = 'Shim.warnAboutValidateProperty';
 
+	/**
+	 * The name of the shim that checks the usage of the deprecated Model::bindModel() method.
+	 *
+	 * @var string
```
Comment (dereuromark) [inline Lib/Shim.php]: those var are not necessary for consts.

Comment (bancer) [reply Lib/Shim.php]: Okay. Eclipse generated them automatically.

Comment (bancer) [reply Lib/Shim.php]: Usually `@var` is used in the core - https://github.com/cakephp/cakephp/search?p=1&q=const&type=&utf8=%E2%9C%93.

Comment (dereuromark) [reply Lib/Shim.php]: "You may use the"
And for const it doesnt really help, as it is clear from the definition what type it is, it cannot change anymore at runtime.

Comment (bancer) [reply Lib/Shim.php]: Of course, it is optional. Although as IDEs add them automatically I would not fight against that.

Comment (dereuromark) [reply Lib/Shim.php]: Not if properly configured.

**Pull request merged**